uyuni/base/uyuni-base.changes

```diff
@@ -1,3 +1,4 @@
+- Do not build uyuni-base server subpackage for client tools
```

Comment:
I think it could be better to  specify which  are  the affected  client tools. this changelog hints  that  we  are not going to build  uyuni-base anymore.

Reply:
It actually means we will not build `uyuni-base-server` package (`server` subpackage of `uyuni-base`) for all of the client tools. Is it fine? Or better to set exact name of the package in the changelog entry?

Reply:
Maybe something like this?

```
Only build `uyuni-base-server` package for server and not for client tools
```

I would vote for having exact package name in the changelog.